### PR TITLE
Use latest @digitalbazaar/vc-status-list.

### PR DESCRIPTION
**Pull request by JSAssassin**

### CODE REVIEW

**Comment by dlongley** [comment]
Do we expect this change to be incompatible with current deployments?

**Comment by JSAssassin** [comment]
@dlongley I marked it as breaking since I thought it won't be compatible with svip status list test suite, is it not breaking?

**Comment by JSAssassin** [comment]
@dlongley only the verifier does since we implemented the status lists tests to verify mock sl credentials. The issuer currently, doesn't issue status list credentials.

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "@digitalbazaar/ed25519-signature-2018": "^2.1.0",
     "@digitalbazaar/ed25519-signature-2020": "^3.0.0",
     "@digitalbazaar/vc": "^2.1.0",
-    "@digitalbazaar/vc-status-list": "^2.1.0",
+    "@digitalbazaar/vc-status-list": "digitalbazaar/vc-status-list#add-positive-test",
```
Comment (dlongley) [inline package.json:27]: Once this is updated I'll approve and merge this. We'll have to figure out what to do wrt. existing deployments elsewhere regarding whether this should be pushed out or not.

Comment (JSAssassin) [reply package.json]: Updated [b7ad589](https://github.com/digitalbazaar/bedrock-vc-verifier/pull/29/commits/b7ad58907ebc313c39fd7585c075c573265354b7)

**Review by dlongley — APPROVED** [review]

**Pull request merged**